README.md

```diff
@@ -118,6 +118,14 @@ envs: |
   </tr>
 </table>
 
+## Action Outputs
+
+`image`: The name of the built image.<br>
+For example, `spring-image`.
+
+`tags`: The list of tags delimeted by a space.<br>
```

Comment:
It's "delimited". Let's just say "A list of the tags that were created, separated by spaces".